app/models/hmis/build_report.rb

```diff
@@ -0,0 +1,66 @@
+module Hmis
```

Comment:
Let's be consistent with the naming, as long and unwieldy as it is and call this module GrdaWarehouse instead of HMIS, which has another meaning in the homeless world and the warehouse.